packages/bun-usockets/src/eventing/libuv.c

```diff
@@ -24,7 +24,7 @@
 /* uv_poll_t->data always (except for most times after calling us_poll_stop)
  * points to the us_poll_t */
 static void poll_cb(uv_poll_t *p, int status, int events) {
-  us_internal_dispatch_ready_poll((struct us_poll_t *)p->data, status < 0,
+  us_internal_dispatch_ready_poll((struct us_poll_t *)p->data, status < 0 && status != UV_EOF, status == UV_EOF,
```

Comment:
Should this be bitwise?
```suggestion
  us_internal_dispatch_ready_poll((struct us_poll_t *)p->data, status < 0 && status & UV_EOF, !(status & UV_EOF),
```

Reply:
I think should not https://docs.libuv.org/en/v1.x/stream.html